### PR TITLE
[6] Product Page using HTML

### PR DESCRIPTION
**Pull request by Cynthia-Berry**
## Task: `[6] Product Page using HTML`
- `This is the page product of my E-commerce website
snapshot1
snapshot2
snapshot3
snapshot 4

`

### CODE REVIEW

**Comment by Cynthia-Berry** [comment]
Hello Team,
Kindly find my PR on Product Page using HTML.
Thank you.

File: submissions/cynthiaNwakaeme/HTML/debug.log [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[0715/181751.412:ERROR:crash_report_database_win.cc(428)] unexpected header
```
Comment (ToluAkin) [inline submissions/cynthiaNwakaeme/HTML/debug.log:1]: You need to remove this file

File: submissions/cynthiaNwakaeme/HTML/index.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                <li><a href="#">&#x1F6D2; My Cart </a></li>
+                <li><a href="#">Help</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section>
+        <main>
+            <span><em>Home > SkinCare > Black+Soap</em></span>
+            <div>
+                <img src="img/PAcked black soap.jpg" alt="J-Royalty African Black Soap" width="30%">
```
Comment (ToluAkin) [inline submissions/cynthiaNwakaeme/HTML/index.html:35]: The require is no styling and not more than 2 product images.

File: submissions/cynthiaNwakaeme/HTML/index.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+                <em><a href="#">More reviews..</a></em>
+            </div>
+        </aside>
+    </section>
+    <section>
+        <hr>
+        <h2>You May like</h2>
+        <ul>
+            <oi>
+                <img src="img/black soap 2.jpg" alt="" width="10%">
```
Comment (ToluAkin) [inline submissions/cynthiaNwakaeme/HTML/index.html:117]: Remove width from all the images

File: submissions/cynthiaNwakaeme/HTML/index.html [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+                <h4>Organic African Soap</h4>
+            </oi>
```
Comment (ToluAkin) [inline submissions/cynthiaNwakaeme/HTML/index.html:119]: <oi> is an invalid html element. i think you are trying to write <ol>

File: submissions/cynthiaNwakaeme/HTML/index.html [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+            <span><em>Home > SkinCare > Black+Soap</em></span>
+            <div>
+                <img src="img/PAcked black soap.jpg" alt="J-Royalty African Black Soap" width="30%">
+                <img src="img/African-Black-Soap.jpg" alt="Unpacked J-Royalty African Black Soap" width="30%">
+            </div>
+            <h2>J-royalty Organic African black soap - 450g</h2> 
+            <span><em>Rating: </em>&#9733;&#9733;&#9733;&#9734;&#9734;</span>
+            <div>
+                <h2>&#x20A6;1,500</h2>
+                <table border="black">
```
Comment (ToluAkin) [inline submissions/cynthiaNwakaeme/HTML/index.html:42]: Also, remove this inline styling for the border to be black

File: submissions/cynthiaNwakaeme/HTML/index.html [hunk 6]
```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/cynthiaNwakaeme/HTML/index.html:1]: You would need to delete the images that you are not using.

File: submissions/cynthiaNwakaeme/HTML/index.html (hunk 4 above)
Comment (Cynthia-Berry) [reply submissions/cynthiaNwakaeme/HTML/index.html:118]: I have effected the changes. kindly review

**Pull request closed without merging**